vortex-file/src/open.rs

```diff
@@ -60,7 +60,7 @@ impl<F: FileType> VortexOpenOptions<F> {
         Self {
             read,
             options,
-            registry: DEFAULT_REGISTRY.clone(),
+            registry: FULL_REGISTRY.clone(),
```

Comment:
Not convinced by this behavior, not sure if `ArrayRegistry::default()` is better here.